blog/_posts/2023-04-18-integrating-scalajs-with-vite.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: blog-detail
+post-type: blog
+by: Sébastien Doeraene
+title: Integrating Scala.js with Vite
```

Comment:
Consider a title that foregrounds the benefit to readers. The gist of the post is "here's how your life is about to get better", not "we built a thing". The thing is a means to an end.